.markdownlint.json

```diff
@@ -0,0 +1,5 @@
+{
+    "default": true,
+    "MD013": false, // 80 char line length
```

Comment:
I would like to include this rule but will enable it with a follow-up PR as there are a lot of violations.